src/test/java/com/github/lombrozo/jsmith/antlr/rules/RulerefTest.java

```diff
@@ -54,4 +55,46 @@ void generatesRuleReferenceByUsingLink() throws WrongPathException {
             Matchers.equalTo(expected)
         );
     }
+
+    @Test
+    void generatesRuleReferenceByUsingArgActionBlock() throws WrongPathException {
+        final String ref = "reference";
+        final ArgActionBlock actionblock = new ArgActionBlock(new Root(), new ArrayList<>(0));
+        final String expected = "Linked rule value(1)";
+        final String expectedtwo = "Linked rule value(2)";
```

Comment:
@RechnoyBobr Can we use a bit more meaningful names? `first`, `second`, `apple`, `orange`, ... anything as a single meaningful noun. (I like `expected` but `expectedtwo` looks a bit strange.)